src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -323,7 +323,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('vm_size', help='The size of virtual machines in the pool. All virtual machines in a pool are the same size.', default='Standard_D2')
         c.argument('vm_image_publisher', help='The publisher of the Azure Virtual Machines Marketplace image.', default='MicrosoftWindowsServer')
         c.argument('vm_image_offer', help='The offer type of the Azure Virtual Machines Marketplace image.', default='WindowsServer')
-        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2019-Datacenter')
+        c.argument('vm_image_sku', help='The SKU of the Azure Virtual Machines Marketplace image.', default='2022-Datacenter')
```

Comment:
I would like to confirm if changing the default values will cause a breaking change in the customer's automated scripts? Can the functions of `2022-Datacenter` fully cover `2019-Datacenter`?

Reply:
no, no shouldn't affect customer experience